src/jit/lsra.cpp

```diff
@@ -8270,7 +8270,7 @@ void LinearScan::resolveRegisters()
             {
                 JITDUMP(" internal");
                 GenTreePtr indNode = nullptr;
-                if (treeNode->OperIsIndir())
+                if (treeNode->OperGet() == GT_IND)
```

Comment:
>              if (treeNode->OperGet() == GT_IND) [](start = 2, length = 48)

If I remember correctly, prior to first class struct work, OperIsIndir() was returning true if oper is GT_IND/STOREIND/NULLCHECK. 
Now this check is reduced to just checking GT_IND.  Don't we need to check here for STOREIND?